utils/Cargo.toml

```diff
@@ -1,12 +1,12 @@
 [package]
+edition = "2021"
 name = "miden-node-utils"
 version = "0.1.0"
```

Comment:
Same comment as before re sorting the `package` section.

Reply:
reverted